views/flow/index.erb

```diff
@@ -0,0 +1,32 @@
+<h1> This is the flow index page </h1>
+
+<% if @id %>
+
+Welcome! Your user id is: <%= @id %>
+
+<% end %>
+
+<br>
+<br>
+
+<% @messages.each do |message| %>
+
+  Message content is:
+  <br>
+  <b> <%= message.content %> </b>
+  <br>
+
+  Message id is:
+  <%= message.id %>
+  <br>
+
+  Message time and date are:
```

Comment:
I think you've done well here by splitting out the time/date and message parts of the peep and then bringing them together here in the view file. It makes for easy-to-read code which is always good!